config.go

```diff
@@ -12,168 +12,200 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//go:generate paramgen -output config_paramgen.go Config
+
 package generator
 
 import (
 	"errors"
 	"fmt"
-	"math"
-	"strconv"
 	"strings"
 	"time"
-)
 
-const (
-	RecordCount   = "recordCount"
-	ReadTime      = "readTime"
-	SleepTime     = "burst.sleepTime"
-	GenerateTime  = "burst.generateTime"
-	FormatType    = "format.type"
-	FormatOptions = "format.options"
-	Operation     = "operation"
-
-	FormatRaw        = "raw"
-	FormatStructured = "structured"
-	FormatFile       = "file"
+	"github.com/conduitio/conduit-connector-generator/internal"
+	sdk "github.com/conduitio/conduit-connector-sdk"
+	"golang.org/x/time/rate"
 )
 
-var (
-	knownFieldTypes = []string{"int", "string", "time", "bool"}
-	requiredFields  = []string{FormatType, FormatOptions}
+const (
+	FormatTypeRaw        = "raw"
+	FormatTypeStructured = "structured"
+	FormatTypeFile       = "file"
 )
 
-type RecordConfig struct {
-	FormatType    string
-	FormatOptions interface{}
-	Operation     string
+type Config struct {
+	Burst ConfigBurst `json:"burst"`
```

Comment:
Why not adding a comment regarding `Burst`?

Reply:
I can add some explanation, just know that this won't show up in the specs, as it contains nested parameters, so it will only be useful for developers.